CHANGELOG.md

```diff
@@ -3,6 +3,127 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [10.0.3-dev.2](https://github.com/Agoric/agoric-sdk/compare/@agoric/sdk@10.0.2...@agoric/sdk@10.0.3-dev.2) (2021-11-24)
```

Comment:
All this stuff goes in the changelog _again_?

Reply:
Yeah, that was during testing.  I've fixed it up now.